charts/ccd-definition-designer-api/values.template.yaml

```diff
@@ -0,0 +1,43 @@
+postgresql:
+  initdbScripts:
+    init.sql: |-
+      CREATE USER ccd WITH PASSWORD 'ccd';
+      CREATE DATABASE ccd_definition_designer WITH OWNER = ccd ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+  nameOverride: postgres
+  persistence:
+    enabled: false
+
+java:
+  image: ${IMAGE_NAME}
+  ingressHost: ${SERVICE_FQDN}
+  applicationPort: 4454
+  environment:
+    DEFINITION_DESIGNER_DB_HOST: ${SERVICE_NAME}-postgres
+    DEFINITION_DESIGNER_DB_PORT: 5432
+    DEFINITION_DESIGNER_DB_NAME: ccd_definition_designer
+    DEFINITION_DESIGNER_DB_USERNAME: ccd
+    DEFINITION_DESIGNER_DB_PASSWORD: ccd
+    DEFINITION_DESIGNER_DB_USE_SSL: false
+    ENABLE_DB_MIGRATE: true
+
+    IDAM_USER_URL: http://betaDevBccidamAppLB.reform.hmcts.net
+    DEFINITION_STORE_IDAM_KEY: ${CCD_DEFINITION_SERVICE_SECRET}
```

Comment:
I need to check the env variable name for the key

Reply:
Most likely, not sure if you need it.